### ISSUE TITLE
Supportn for IPv6 addresses

### ISSUE THREAD

**Issue by sp35** [intelowlproject/IntelOwl-ng#126]
![image](https://user-images.githubusercontent.com/51958314/128022761-70156d4c-6e88-4be6-8ea8-c44731120d2b.png)

**Comment by mlodic** [comment]
Yeah but it is not used a lot so I guess it is something of low priority

**Comment by Eshaan7** [comment]
Just need to adjust the regex to allow both v4 and v6. Same applies for PyIntelOwl.
